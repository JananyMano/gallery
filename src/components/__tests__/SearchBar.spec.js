import React from "react";
import { create, act } from "react-test-renderer";
import SearchBar from "../SearchBar";

describe("Search bar component", () => {
  test("Search bar component matches the snapshot", () => {
    const searchbar = create(<SearchBar />);
    expect(searchbar.toJSON()).toMatchSnapshot();
  });

});