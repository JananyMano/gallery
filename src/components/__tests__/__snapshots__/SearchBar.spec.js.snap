// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Search bar component Search bar component matches the snapshot 1`] = `
<div
  className="ui segment"
>
  <form
    className="ui form"
    onSubmit={[Function]}
  >
    <div
      className="field"
    >
      <label>
        Image Search
      </label>
      <input
        onChange={[Function]}
        type="text"
        value=""
      />
    </div>
  </form>
</div>
`;
